libs/island-ui/core/src/lib/DatePicker/overwriteStyles.css

```diff
@@ -0,0 +1,100 @@
+.react-datepicker {
```

Comment:
is there no other way to overwrite the styles apart from using this css file? It would be optimal to do it with treat (not that I know how, maybe @DavidBachmann has ideas). All consumers of this component now need to have css loader to use it. I understand it may be inevitable, just wondering if there are other possibilities

Reply:
Not that I know of 😕 @DavidBachmann, do you have any ideas?

Reply:
Here's half the conversion done. Could you convert this to use our theme?

```jsx
    <div className={styles.root}>
      <ReactDatePicker ... />
     </div>
```

```js
import { style, globalStyle } from 'treat'

export const root = style({
  position: 'relative',
})

globalStyle(`${root} .react-datepicker`, {
  display: 'block',
  borderTopRightRadius: '0',
  borderTopLeftRadius: '0',
  boxShadow: '-4px 4px 0px #00e4ca, 4px 4px 0px #00e4ca',
  border: 'none',
  padding: '16px 24px',
})

globalStyle(`${root} .react-datepicker-wrapper`, { display: 'block' })

globalStyle(`${root} .react-datepicker__header`, {
  backgroundColor: 'white',
  borderBottom: 'none',
})
globalStyle(`${root} .react-datepicker-popper[data-placement^='bottom']`, {
  marginTop: '0',
  right: '0',
  top: '68px !important',
  transform: 'none !important',
})
globalStyle(`${root} .react-datepicker__month-container`, { float: 'none' })
globalStyle(`${root} .react-datepicker__month`, { margin: '0 0 16px 0' })
globalStyle(`${root} .react-datepicker__day-names`, { marginBottom: '4px' })
globalStyle(`${root} .react-datepicker__day-name, .react-datepicker__day`, {
  fontFamily: 'IBM Plex Sans',
  fontStyle: 'normal',
  fontWeight: 600,
  fontSize: '18px',
  lineHeight: '28px',
  textAlign: 'center',
  color: '#00003c',
  margin: '0 28px 0 0',
})

globalStyle(
  `${root} .react-datepicker__day-name:last-child, .react-datepicker__day:last-child`,
  {
    margin: '0',
  },
)

globalStyle(`${root} .react-datepicker__day--disabled`, {
  color: '#ccccd8 !important',
})

globalStyle(`${root} .react-datepicker__day--disabled:hover`, {
  borderColor: 'transparent !important',
})

globalStyle(`${root} .react-datepicker__day`, {
  fontFamily: 'IBM Plex Sans',
  fontStyle: 'normal',
  fontWeight: 300,
  fontSize: '18px',
  lineHeight: '28px',
  textAlign: 'center',
  color: '#00003c',
  border: '1px solid transparent',
  borderRadius: '5px',
  outline: 'none !important',
  transition: 'border-color 0.3s, color 0.3s',
})

globalStyle(`${root} .react-datepicker__day:hover`, {
  background: '#f2f7ff',
  borderColor: '#0061ff',
})

globalStyle(
  `${root} .react-datepicker__day--selected, .react-datepicker__day--selected:hover, .react-datepicker__month-text:hover, .react-datepicker__quarter-text:hover, .react-datepicker__year-text:hover, .react-datepicker__day--keyboard-selected`,
  {
    background: '#0061ff !important',
    color: 'white !important',
  },
)

```

Reply:
I see now that it doesn't matter if we overwrite these styles in Treat or not because we are importing a css file with the styles from React DatePicker (see https://github.com/island-is/island.is/pull/670/files#diff-d9baf8f84ac33bde02df45cd9c60bb56R17) and now the build for air-discount-scheme-web and gjafakort-web are failing because of this error.

![Screenshot 2020-09-07 at 14 59 57](https://user-images.githubusercontent.com/3789875/92400113-e1ebeb80-f11a-11ea-8163-cc9292f841a8.png)

How do we want to remedy this, @DavidBachmann @ArniFreyrSnorrason? 🤔

Reply:
IMO we do it the same as above. It's painful but much better than importing third party CSS.

Reply:
Or maybe not!? Their CSS is like 900 lines... Let me think about this

Reply:
If we want to get this merged ASAP we could let DatePicker come with a disclaimer that the App **must** import that style or it won't display correctly.

Reply:
It was decided **not** to have these styles in every App component. I have moved it into `react-datepicker.treat.ts`. Specificity is tricky so the overwrites got a `island-ui-datepicker` className to boost the specificity score. Let me know if you have a better idea.